scripts/hooks/clangFormatFunc

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+endWith='\.(cpp|cc|h|hpp)$'
+ignoreFolder='/(PerfApp|android|ios|nodejs|vscode|community|dotnet)/'
+
+function checkFormat() {
+	if [ -z "$clangFormat" ]; then
+		echo '$clangFormat is not set correct in clangFormatCheck'
+		return 1
+	fi
```

Comment:
just invoke `clang-format` directly -- if we've gotten this far, it should be available